django_project/frontend/views/switch_organisation.py

```diff
@@ -26,7 +27,11 @@ def switch_organisation(request, organisation_id):
             user=request.user,
             organisation__id=organisation_id
         )
-        if not organisation_user.exists():
+        organisation_rep = OrganisationRepresentative.objects.filter(
```

Comment:
Is this required since we will also have record in OrganisationUser for a manager?

Reply:
@danangmassandy No, I've updated it to check only from OrganisationUser.